### ISSUE TITLE
Dead or unnecessary code

### ISSUE THREAD

**Issue by thomasfr** [cesardeazevedo/sails-hook-sequelize-blueprints#6]
Is it possible that there is some dead code which gets not executed due to copy&paste from core blueprints hook? For instance: https://github.com/cesardeazevedo/sails-hook-sequelize-blueprints/blob/master/actions/add.js#L111 - Because pubsub has to be disabled anyways.
Or this file: https://github.com/cesardeazevedo/sails-hook-sequelize-blueprints/blob/master/jsonp.js
Is that correct?
If you want i can remove that lines and files.

**Comment by cesardeazevedo** [comment]
Yes, it's a fork from the original sails blueprints, which there's a lot of dead code as well (https://github.com/balderdashy/sails/blob/master/lib/hooks/blueprints/jsonp.js)

fell free to remove it.

About pubsub, i haven't tested it yet, can leave it there for now.

Thank you.
